### ISSUE TITLE
Do you want to archive this repo?

### ISSUE THREAD

**Issue by Garfield550** [PFCKrutonium/Windows-10-Login-Background-Changer#307]
It's a suggestion.

**Comment by Krutonium** [comment]
That might be a good idea.

**Comment by darth62969** [comment]
good idea... but i rather leave it up because it's HISTORY... or well,
incase MS does something or someone decides to work on it again, such that
we can have 2 different wallpapers on the lock screen... something i still
want...
(: Jonathan :)

**Comment by Krutonium** [comment]
I mean, we can always un-archive it, if need be.
